### ISSUE TITLE
Create `.keepme` files in directories after creation/`brew update`

### ISSUE THREAD

**Issue by MikeMcQuaid** [Homebrew/brew#6263]
# A detailed description of the proposed feature

The top-level directories in `/usr/local` e.g. `/usr/local/sbin` should have a `.keepme` file or similar created in them to avoid their removal in the macOS Catalina 10.15 installer.

# The motivation for the feature

Avoid users having to manually `sudo` to recreate these directories.

# How the feature would be relevant to at least 90% of Homebrew users

Most Homebrew users will end up upgrading to 10.15.

# What alternatives to the feature have been considered

Requiring users to manually recreate these.

**Comment by veera83372** [comment]
Can I pick this one ?

**Comment by reitermarkus** [comment]
@veera83372, go ahead! 👍

**Comment by idoo** [comment]
added PR https://github.com/Homebrew/brew/pull/6464

**Comment by MikeMcQuaid** [comment]
Fixed in https://github.com/Homebrew/brew/pull/6464

**Comment by KizzyCode** [comment]
Hi, do you have any further information or a reference by Apple how this file is honored? Asking because I'm already using Catalina Beta 8 and the folders haven't been removed yet even without this file. It could be relevant to some of my projects but I've tried to search for and was unable to find any sources.

**Comment by MikeMcQuaid** [comment]
@KizzyCode I don't have a public source, sorry. I believe @fxcoudert saw this during testing and we were advised by Apple engineers to do this.

**Comment by zhao-li** [comment]
This is the only thread I found regarding this issue.  Sorry if I'm posting in the wrong place.

I think `brew update` is adding this file: `/usr/loca/sbin/.keepme`
and then `brew doctor` gives this warning:
```
Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting the PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.bash_profile```

Everything works, but I was wondering if there's a way to avoid the extraneous warning without breaking something else in brew.

Thank you for your time.

**Comment by simono** [comment]
@zhao-li I fixed this in #6568 :)

**Comment by zhao-li** [comment]
Yay! Thanks😁
